x/globalfee/module.go

```diff
@@ -119,8 +124,15 @@ func (a AppModule) RegisterInvariants(registry sdk.InvariantRegistry) {
 // 	return nil
 // }
 
-func (a AppModule) RegisterServices(cfg module.Configurator) {
-	types.RegisterQueryServer(cfg.QueryServer(), NewGrpcQuerier(a.paramSpace))
+func (am AppModule) RegisterServices(cfg module.Configurator) {
```

Comment:
why need to migrate? can't we use new params directly? since globalfee is not live on chain yet.